flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/CompletedCheckpointStats.java

```diff
@@ -63,6 +93,7 @@
      * @param totalSubtaskCount Total number of subtasks for the checkpoint.
      * @param taskStats Task stats for each involved operator.
      * @param numAcknowledgedSubtasks Number of acknowledged subtasks.
+     * @param checkpointedSize Total checkpointed size over all subtasks.
      * @param stateSize Total checkpoint state size over all subtasks.
```

Comment:
I think it makes sense to update the javadoc for `stateSize` (ditto other stats classes).

Reply:
How about make `checkpointedSize` as `Total persisted data size over all subtasks of this checkpoint.` and leave `stateSize` as before?

Reply:
Maybe it would be enough to mention sync and async phases as per discussion [above](https://github.com/apache/flink/pull/18324#discussion_r790718434).